packages/react-components/react-toolbar/src/components/ToolbarRadio/ToolbarRadio.tsx

```diff
@@ -1,18 +1,18 @@
 import * as React from 'react';
 import type { ToolbarRadioProps } from './ToolbarRadio.types';
 import type { ForwardRefComponent } from '@fluentui/react-utilities';
-import { useRadio_unstable, renderRadio_unstable } from '@fluentui/react-radio';
+import { renderToggleButton_unstable } from '@fluentui/react-button';
+import { useToolbarRadio_unstable } from './useToolbarRadio';
 import { useToolbarRadioStyles_unstable } from './useToolbarRadioStyles';
-import { useToolbarContext_unstable } from '../Toolbar/ToolbarContext';
 
 /**
- * ToolbarRadio component is a Radio to be used inside Toolbar
+ * ToolbarToggleButton component
  */
 export const ToolbarRadio: ForwardRefComponent<ToolbarRadioProps> = React.forwardRef((props, ref) => {
```

Comment:
How about let's call this `ToolbarRadioButton` to be consistent with `ToolbarToggleButton`

Reply:
Done https://github.com/microsoft/fluentui/pull/25343/commits/886d59ea883d7ea23d458de463ae92a54a745f2e